MeshLib/Mesh.h

```diff
@@ -125,6 +125,16 @@ class Mesh : BaseLib::Counter<Mesh>
     MeshLib::Properties & getProperties() { return _properties; }
     MeshLib::Properties const& getProperties() const { return _properties; }
 
+    bool isAxiallySymmetric() const { return _is_axially_symmetric; }
+    void setAxiallySymmetric(bool is_axial_symmetric) {
```

Comment:
I excluded it, because no interesting 1D axially symmetric problem came to my mind. Do you want 1D, too?